pkg/graveler/graveler_test.go

```diff
@@ -1030,6 +1030,9 @@ func testGravelerCommit(t *testing.T, kvEnabled bool) {
 	expectedCommitID := graveler.CommitID("expectedCommitId")
```

Comment:
The failing flows are also really important to validate: a sealed token was added and a staging token was replaced -> you need to make sure that the branch is in the expected state.

Reply:
I'll add it after you'll add the mocks